lib/macos.js

```diff
@@ -45,6 +48,7 @@ var MacosBuilder = {
     options.log( 'Wrote temporary ´appdmg.json´' );
     options.log( 'Kicking off ´appdmg´' );
 
+    var appdmg = require( 'appdmg' );
     var ee = appdmg( {
```

Comment:
Sure, this package does not install successfully on windows, which is why I moved it into optionalDependencies. So when you require it on windows/linux it will throw. This module always gets loaded so it's not safe to require it always, only attempt to require it when you are building for osx and then you don't get a throw.